### PR TITLE
Add profile.plugin

### PR DESCRIPTION
**Pull request by vkalintiris**
##### Summary

A small plugin that allows someone to backfill the agent with random data.

##### Test Plan

See `collectors/profile.plugin/README.md`

##### Additional Information

This is meant for devs and users who want to get an idea of that the worst-case scenario looks like. The generated data are random (ie. not amenable to compression) and all the charts/dims are attached to localhost (ie. the various indexes are stress-tested).

### CODE REVIEW

File: collectors/profile.plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# profile.plugin
+
+This plugin allows someone to backfill an agent with random data.
+
+A use can specify:
```
Comment (MrZammler) [inline collectors/profile.plugin/README.md:5]: Just a small typo `use` -> `user` ?

**Comment by andrewm4894** [comment]
I think this could actually be quite useful for users looking to figure out sizings and things like that. @sashwathn as fyi

**Comment by sashwathn** [comment]
@andrewm4894 : Thanks for tagging me here. We will need to check how this can be digested by our customers?

**Review by thiagoftsm — APPROVED** [review]
PR is working as expected, LGTM!

**Comment by vkalintiris** [comment]
> We will need to check how this can be digested by our customers?

Yes, but in order to do this we need to truly understand what kind of discrepancies we should expect against a real setup.

I've tested this with charts/dimensions corresponding to 16 real agents. I was able to backfill 7 days of data in ~2 hours. Ideally, I'd like to setup a real parent with 16 real children streaming to it and let it run for a week and investigate any differences.

**Review by thiagoftsm — APPROVED** [review]
I retested after rebase and I review last commits, everything is working as expected, LGTM!

File: collectors/profile.plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  number of threads = 16
+```
+
+The `number of threads` option will create the specified number of collection
+threads. The rest of the options apply to each thread individually, eg. the
+above configuration will create 3200 charts, 16000 dimensions in total, which will be
+backfilled for the duration of 1 day.
+
+Note that all but the 1st chart created in each thread will be marked as hidden
+in order to ease the load on the dashboard's UI.
```
Comment (MrZammler) [inline collectors/profile.plugin/README.md:34]: Hi Vasili. Just a small note that unless the plugin is enabled we won't have these options in `netdata.conf`. Not sure if we should add this in the README, or perhaps expose the options even if the plugin is disabled?

Of course for old users, the section won't appear in their netdata.conf at all, so in general we could say something like "add these options to control the plugin." or similar.

**Review by MrZammler — APPROVED** [review]
Other than maybe a small readme change, it looks good to me!

File: collectors/profile.plugin/README.md (hunk 2 above)
Comment (vkalintiris) [reply collectors/profile.plugin/README.md:34]: You're right. I don't want to go through another 2nd round of approvals just for that.

**Pull request merged**